localization/localization_common/scripts/localization_common/utilities.py

```diff
@@ -73,11 +73,40 @@ def load_dataframe(files):
     return dataframe
 
 
-def run_command_and_save_output(command, output_filename, print_command=True):
+def run_command_and_save_output(command, output_filename="", print_command=True):
```

Comment:
Nice! Does this now alert the user when the script fails? Can you add some comments to the code blocks?

Reply:
Added comments to the code. Yes it alerts the user by printing the error message and exits the program